sw/vendor/riscv_compliance/riscv-target/opentitan/README.md

```diff
@@ -0,0 +1,135 @@
+---
+title: "OpenTitan RISC-V Compliance Testing"
+---
+
+# Overview
+The RISC-V compliance test can be run on either FPGA or verilator.
+To run on verilator, set the variables below
+
+```console
+$ export RISCV_TARGET=opentitan
+$ export RISCV_DEVICE=rv32imc
+$ export TARGET=verilator
+```
+
+To run on FPGA, set the variables below.
+The `FPGA_UART` variable must be set to wherever a valid device is connected.
+
+```console
+$ export RISCV_TARGET=opentitan
+$ export RISCV_DEVICE=rv32imc
+$ export TARGET=fpga
+$ export FPGA_UART=/dev/tty*
+```
+
+By default, the test assumes there exists a valid verilator build at `${REPO_TOP}/build/lowrisc_systems_top_earlgrey_verilator_0.1/sim-verilator/Vtop_earlgrey_verilator`.
+If your verilator build is at a different location, please set that as well when running with verilator.
+For instructions on how to create a verilator build, please refer to the [verilator guide]({{< relref "doc/ug/getting_started_verilator" >}}).
+
+```console
+$ export TARGET_SIM=${PATH_TO_VERILATOR_BUILD}
+```
+
+When running against FPGA, the test assumes the FPGA is already programmed and ready to go.
+To find out how to properly build and flash FPGA, please refer to the [FPGA manual]({{< relref "doc/rm/ref_manual_fpga" >}})
+
+
+Now, run the tests.
+The following output will be seen (software build steps are truncated).
+The example below uses verilator as an example, but the FPGA output is nearly identical.
+
+```console
+$ cd $REPO_TOP/sw/vendor/riscv_compliance
+$ make RISCV_ISA=rv32i && make RISCV_ISA=rv32im && make RISCV_ISA=rv32imc
+
+
+
+Rom initialized with program at /usr/local/google/home/timothytim/ci2_dev/ot_core/otpriv2/public/sw/vendor/riscv_compliance/../../boot_rom/rom.vmem
```

Comment:
I would use a generic path (like `$REPO_TOP/sw/vendor/...`), rather than a path on your workstation.

Reply:
oops, thanks for catching that.  I forgot to replace it.

Reply:
o actually i just realized this was output i directly copied...